frontend/config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  SERVER_PORT: 8000,
+  IS_DEBUG: (process.env.NODE_ENV === 'development'),
+
+  // TODO: Move addon build to a better path
+  ADDON_SRC_PATH: './addon/',
+
+  SRC_PATH: './frontend/src/',
+  DEST_PATH: './frontend/build/',
+  DIST_PATH: './dist/',
+  DJANGO_DIST_PATH: './testpilot/frontend/static/',
+  CONTENT_SRC_PATH: './content-src/',
+
+  PRODUCTION_EXPERIMENTS_URL: 'https://testpilot.firefox.com/api/experiments',
+  IMAGE_NEW_BASE_PATH: 'images/experiments/',
+  IMAGE_NEW_BASE_URL: '/static/images/experiments/',
+
+  'sass-lint': true,
+  'js-lint': true
+};
+
+// Pull in local debug-config.json overrides
+const tryRequire = require('try-require');
```

Comment:
Sorry, for some reason my computer was missing the closing `)`, or else my eyes are starting to go. But it looks fine now, so I'll delete these useless and now confusing comments.